src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -432,6 +446,17 @@ public List<String> run() {
         }
     }
 
+    private void verifyTLSVersion(final String settings, final List<String> configuredProtocols) {
+        for (final var tls : configuredProtocols) {
+            if (tls.equalsIgnoreCase("TLSv1.1") || tls.equalsIgnoreCase("TLSv1")) {
+                deprecationLogger.deprecate(
```

Comment:
Is this being called with the right args? The other instances I see of this being called have args settingKey, message, then list of args to replace any placeholders in the message.

Other instances of deprecationLogger.deprecate: https://github.com/opensearch-project/security/blob/b7b49b9de147126a7c7a3ed0f2f1a33d02eaee9f/src/main/java/org/opensearch/security/setting/DeprecatedSettings.java#L23-L29

DeprecationLogger.deprecate declaration: https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/common/logging/DeprecationLogger.java#L102-L108

Reply:
good point. thank you

Reply:
Fixed